File: baekjoon/p02903/Main.java
package baekjoon.p02903;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.nextInt()));
    }

    private static int solve(int n) {
        int x = 2;
        while (n-- > 0) {
            x = (x * 2 - 1);
        }

        return (x * x);
    }

}

File: baekjoon/p02903/Main.kt
package baekjoon.p02903

fun main() {
    println(solve(readLine()!!.toInt()))
}

private fun solve(n: Int): Int {
    var x = 2
    repeat(n) {
        x = (x * 2 - 1)
    }

    return x * x
}